content/en/docs/started/installing-kubeflow.md

```diff
@@ -9,140 +9,190 @@ Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it prov
 Operators can choose what is best for their users, there is no requirement to deploy every component.
 To read more about the components and architecture of Kubeflow, please see the <a href="/docs/started/architecture/">Kubeflow Architecture</a> page.
 
+### How to install Kubeflow?
+
 There are two pathways to get up and running with Kubeflow, you may either:
-1. Use a [packaged distribution](#packaged-distributions)
-1. Use the [manifests](#manifests) (advanced)
+
+1. Use a [Packaged Distributions of Kubeflow](#packaged-distributions-of-kubeflow)
+2. Use the [Raw Kubeflow Manifests](#raw-kubeflow-manifests) (advanced)
 
 <a id="packaged-distributions"></a>
-## Install a packaged Kubeflow distribution
+<a id="install-a-packaged-kubeflow-distribution"></a>
+## Packaged Distributions of Kubeflow 
 
 {{% alert color="warning" %}}
 Packaged distributions are developed and supported by their respective maintainers, the Kubeflow community does not currently endorse or certify any distribution.
 {{% /alert %}}
 
-<b>See the table below for a list of options and links to documentation:</b>
+### Active Distributions
+
+The following table lists some <b>distributions that provide <a href="/docs/releases/kubeflow-1.6/"><code>Kubeflow 1.6</code></a> or later</b>.
 
 <div class="table-responsive">
   <table class="table table-bordered">
     <thead class="thead-light">
       <tr>
         <th>Name</th>
         <th>Maintainer</th>
-        <th>Platform</th>
-        <th>Version</th>
-        <th>Docs</th>
-        <th>Website</th>
+        <th>Target Platform</th>
+        <th>Links</th>
       </tr>
     </thead>
     <tbody>
       <tr>
         <td>Kubeflow on AWS</td>
-        <td>Amazon Web Services (AWS)</td>
-        <td>Amazon Elastic Kubernetes Service (EKS)</td>
-        <td>{{% aws/latest-version %}}</td>
-        <td><a href="/docs/distributions/aws/">Docs</a></td>
-        <td><a href="https://awslabs.github.io/kubeflow-manifests">External Website</a></td>
-      </tr>
-      <tr>
-        <td>Kubeflow on Azure</td>
-        <td>Microsoft Azure</td>
-        <td>Azure Kubernetes Service (AKS)</td>
-        <td>1.2</td>
-        <td><a href="/docs/distributions/azure/">Docs</a></td>
-        <td></td>
+        <td>Amazon Web Services</td>
+        <td>
+          ➤ Amazon Elastic Kubernetes Service (EKS)
```

Comment:
nit: ➤ symbol can be mistaken as a clickable drop-down

Only Arrikto has three listed, can we remove this and have bullet points for Arrikto only?

Reply:
@annajung I agree that the  `➤` symbol was confusing (and messy).

I have removed the "target platform" bullets for all but the Arrikto one and used a smaller bullet `◦` for that case.

I have updated the "links" bullets to use `⊛` for the "official website" and `◌` for the "other docs".